sympy/core/add.py

```diff
@@ -595,10 +595,13 @@ def _eval_is_nonnegative(self):
         from sympy.core.exprtools import _monotonic_sign
         if not self.is_number:
             c, a = self.as_coeff_Add()
-            if not c.is_zero and a.is_nonnegative:
+            if not c.is_zero:
                 v = _monotonic_sign(a)
```

Comment:
I don't understand why c is pulled out before passing the expression to this function. What happens if you just use `_monotonic_sign(self)`?